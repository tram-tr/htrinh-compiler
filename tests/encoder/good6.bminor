"Form Feed: \\f\nCarriage Return: \\r"