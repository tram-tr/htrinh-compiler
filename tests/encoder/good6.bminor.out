decoded: Form Feed: \f
Carriage Return: \r
encoded: "Form Feed: \\f\nCarriage Return: \\r"
encoding successful.
